### PR TITLE
Add metrics-server for Kubernetes HPA / stats

### PR DESCRIPTION
**Pull request by alexellis**
Tested with "install.sh" against Civo k3s

Signed-off-by: Alex Ellis (OpenFaaS Ltd) <alexellis2@gmail.com>

### CODE REVIEW

File: metrics-server/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: metrics-server
```
Comment (andyjeffries) [inline metrics-server/manifest.yaml]: Please could you name this as Metrics Server, the name is used as a title in display on the site.

File: metrics-server/post_install.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+## metrics-server - provide metrics for Kubernetes HPAv2 (Pod auto-scaling)
+
+### Test it out
+
+* Check that data is being received from nodes:
```
Comment (andyjeffries) [inline metrics-server/post_install.md]: Would rather drop these bullet points, normal paragraphs work well when interspersed with code snippets. If you like I can change these after merging though.

File: metrics-server/manifest.yaml (hunk 1 above)
Comment (alexellis) [reply metrics-server/manifest.yaml]: Sure

File: metrics-server/post_install.md (hunk 2 above)
Comment (alexellis) [reply metrics-server/post_install.md]: Resolved

**Pull request merged**